shinylive/_utils.py

```diff
@@ -59,18 +61,16 @@ def listdir_recursive(dir: str | Path) -> list[str]:
     return all_files
 
 
-FromTo = Tuple[str, str]
-
-
 def copy_file_and_substitute(
-    src: str | Path, dest: str | Path, *, replacements: Sequence[FromTo]
+    src: str | Path,
+    dest: str | Path,
+    **kwargs: object,
```

Comment:
We should try to eliminate collisions when possible. 

```suggestion
    *,
    template_params: dict[str, object],
```